config/routes/overrides.rb

```diff
@@ -1,6 +1,21 @@
+class WhitelistConstraint
+  def initialize
+    @path = /^\/api\/v2\/organizations\/\S+\/parameters/
```

Comment:
After chatting with eric, i think he is suggesting that either we rename the class to  OrganizationWhitelist, or change it to support multiple regex's so that we can use it for any other route we may want to do something similar to